controllers/host/host_controller.go

```diff
@@ -1564,106 +1564,124 @@ func (r *HostReconciler) StopAfterInSync() bool {
 // ReconcileExistingHost is responsible for dealing with the provisioning of an
 // existing host.
 func (r *HostReconciler) ReconcileExistingHost(client *gophercloud.ServiceClient, instance *starlingxv1.Host, profile *starlingxv1.HostProfileSpec, host *hosts.Host, reqNs string) error {
+	logHost.Info("starting reconciliation for existing host", "host", host.ID)
+
 	var defaults *starlingxv1.HostProfileSpec
 	var current *starlingxv1.HostProfileSpec
 	var platform_network_subreconciler_errs []error
+
 	if !host.Stable() {
 		msg := "waiting for a stable state for existing host"
+		logHost.Info("host is not stable", "host", host.ID)
```

Comment:
The msg will be recorded in the StartMonitor, why do we need another similar message?

Reply:
Removed all logs that are printed in StartMonitor